databases/week2/2hyf_lesson2.sql

```diff
@@ -0,0 +1,58 @@
+--db week2 hw
```

Comment:
its always important to mention which schema/database you are going to use, ex:
use hyf_lesson1;